README.md

```diff
@@ -16,7 +16,7 @@ You can learn more about each of these components within the [Quick Start Guide]
 
 ## To Use
 
-To clone and run this repository you'll need [Git](https://git-scm.com) and [Node.js](https://nodejs.org/en/download/) (which comes with [npm](http://npmjs.com)) installed on your computer. From your command line:
+To clone and run this repository you'll need [Git](https://git-scm.com) and [Node.js](https://nodejs.org/en/download/) (which comes with [npm](http://npmjs.com)) installed on your computer. Please fork this repository to your own GitHub account then from your command line:
```

Comment:
Confused by this, the steps below indicate you should clone from `electron/electron-quick-start`, am I missing why users need to fork this repository first?

Reply:
If a person wishes to contribute to the project, they must fork the repo first rather than simply cloning it. I guess the README is simply explaining how to run the app. I was hoping to provide a little extra clarity to the documentation.

Reply:
@prestondoris in that case i would make it explicit that the steps to _contributing_ include forking the repo, but that forking is not necessary in order to simply run the app. Perhaps add that in a slightly separate section?